app/views/content/_page_title.html.erb

```diff
@@ -5,4 +5,4 @@
     'gtm-item-document-type': item.raw_document_type
   } %>
 <br>
-<span class="base-path">/<%= item.base_path %></span>
+<span class="base-path">/<%= CGI::unescape(item.base_path) %></span>
```

Comment:
Not sure about scaping them in our application. The `base_paths` seems to be broken for the Content Items in `PublishingAPI` as well. 

I would rather fixing them in `PubishingAPI`?

Any thoughts @chao-xian @cbaines?

Reply:
I'm unsure what goes on in the Publishing API, but we can check. Even if the Publishing API sends encoded URLs, it's probably better to decode them all as part of the ETL process, rather than in the view.